packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -111,6 +111,13 @@
 				/>
 				<KtFormControllerObject formKey="additionalProps" v-if="componentDefinition.additionalProps.length > 0">
 					<h4>Additional Props</h4>
+					<KtFieldToggle
+						v-if="componentDefinition.additionalProps.includes('actions')"
+						formKey="actions"
+						isOptional
+						label="create new"
```

Comment:
Should be reflective of the prop and may or may not need a `helpText` depending on if it’s obvious what’s happening with the label